roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
```

Comment:
I would distinguish the next few years from the 12 week programme. 

I see somebody being `autonomous` if they are able to run the day-to-day of the academy alone without needing a load of input from directors / wider company. For example, coming in on a Monday, with a POA for the week, getting the standups happening, making decisions about direction, course correcting where necessary, taking people through the learning materials, helping people to succeed, organising showcases, helping academy members to prep showcases etc. 

Obviously feedback and input should be welcomed and will be provided (as part of CF and maybe though Bizops), but being able to run with the academy autonomously as an individual is important.